sonic-thermalctld/scripts/thermalctld

```diff
@@ -629,11 +630,9 @@ class ThermalControlDaemon(daemon_base.DaemonBase):
         """
         if sig == signal.SIGHUP:
             self.log_info("Caught SIGHUP - ignoring...")
-        elif sig == signal.SIGINT:
-            self.log_info("Caught SIGINT - exiting...")
-            self.stop_event.set()
-        elif sig == signal.SIGTERM:
-            self.log_info("Caught SIGTERM - exiting...")
+        elif sig == signal.SIGINT or sig == signal.SIGTERM:
+            self.log_info("Caught signal {} - exiting...".format(sig))
+            self.exit_code = sig
```

Comment:
Suggest changing exit code to `(128 + sig)`, as this is a standard convention in Linux. This was it will also remain consistent with the default signal handler in DaemonBase: https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/daemon_base.py#L53